[2024-03-05T10:49:25.312+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag_v4.second_task manual__2024-03-05T10:49:23.516740+00:00 [queued]>
[2024-03-05T10:49:25.317+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag_v4.second_task manual__2024-03-05T10:49:23.516740+00:00 [queued]>
[2024-03-05T10:49:25.318+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-05T10:49:25.325+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2024-03-05 10:49:23.516740+00:00
[2024-03-05T10:49:25.328+0000] {standard_task_runner.py:60} INFO - Started process 2064 to run task
[2024-03-05T10:49:25.331+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_dag_v4', 'second_task', 'manual__2024-03-05T10:49:23.516740+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpo_pxzuty']
[2024-03-05T10:49:25.332+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask second_task
[2024-03-05T10:49:25.362+0000] {task_command.py:423} INFO - Running <TaskInstance: test_dag_v4.second_task manual__2024-03-05T10:49:23.516740+00:00 [running]> on host a94d9406bb00
[2024-03-05T10:49:25.513+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minh' AIRFLOW_CTX_DAG_ID='test_dag_v4' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-05T10:49:23.516740+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-05T10:49:23.516740+00:00'
[2024-03-05T10:49:25.514+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-05T10:49:25.515+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo this is the second task']
[2024-03-05T10:49:25.520+0000] {subprocess.py:86} INFO - Output:
[2024-03-05T10:49:25.521+0000] {subprocess.py:93} INFO - this is the second task
[2024-03-05T10:49:25.521+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-05T10:49:25.539+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=test_dag_v4, task_id=second_task, execution_date=20240305T104923, start_date=20240305T104925, end_date=20240305T104925
[2024-03-05T10:49:25.560+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-05T10:49:25.571+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
